### PR TITLE
User Class 

### PR DESCRIPTION
**Pull request by jdvolk**
#### What's this PR do?
add properties for user class/ wrote test to check if it is a function and takes in user data/ still working on null if not passed any data test
#### Where should the reviewer start?
src/User.js and tests/User-test.js
#### How should this be manually tested?
run npm test tests/...
#### Any background context you want to provide?
need to figure out better way to test if nothing is passed into the class
#### Questions:
maybe you have a better idea for testing this

### CODE REVIEW

File: src/User.js [hunk 1]
```diff
@@ -1,6 +1,21 @@
 class User {
-  constructor() {
-    
+  constructor(userDetails) {
```
Comment (attom2) [inline src/User.js:2]: maybe if we pass in something that isn't an object it shouldn't set any properties
